front-end-node/NodeInspectorOverrides.js

```diff
@@ -134,3 +150,47 @@ function getAllUiSourceCodes() {
     return p.name();
   }
 }
+
+var notifications = [];
+
+function processDebuggerPaused(event) {
```

Comment:
At current time `processDebuggerPaused` exposed in global scope, will be nice to remove it from global.
There are two solutions for this:
1. Wrap `NodeInspectorOverrides` content by self calling function `(function() {/*...*/} ())`.
2. Define `processDebuggerPaused` and `notifications` as private methods of `NodeInspectorOverrides`

I'd like the second solution. It is more customisable from plugins in future.